src/drivers/lis3mdl/lis3mdl.cpp

```diff
@@ -1178,38 +1178,38 @@ int LIS3MDL::calibrate(struct file *filp, unsigned enable)
 
 int LIS3MDL::check_scale()
```

Comment:
Isn't that equivalent? Same in [HMC5883](https://github.com/PX4/Firmware/blob/master/src/drivers/hmc5883/hmc5883.cpp#L1253-L1269) and [IST8310](https://github.com/PX4/Firmware/blob/master/src/drivers/ist8310/ist8310.cpp#L1193-L1209) Should I change all of them?